api-reference/timescaledb/hypercore/index.mdx

```diff
@@ -0,0 +1,128 @@
+         timescaledb.orderby='time DESC'
+       );
+       ```
+       <CreateHypertableProcedure />
+
+    * **{CAGG}s**
+        1.  [Use `ALTER MATERIALIZED VIEW` for a {CAGG}][compression_continuous-aggregate]:
+             ```sql
+             ALTER MATERIALIZED VIEW assets_candlestick_daily set (
+                timescaledb.enable_columnstore = true,
+                timescaledb.segmentby = 'symbol');
+             ```
+         Before you say `huh`, a {CAGG} is a specialized {HYPERTABLE}.
+
+        1. Add a policy to convert {CHUNK}s to the {COLUMNSTORE} at a specific time interval:
+
+           Create a [columnstore_policy][add_columnstore_policy] that automatically converts {CHUNK}s in a {HYPERTABLE}
+           to
+           the {COLUMNSTORE} at a specific time interval. For example:
+           ``` sql
+           CALL add_columnstore_policy('assets_candlestick_daily', after => INTERVAL '1d');
+           ```
+
+   {TIMESCALE_DB} is optimized for fast updates on compressed data in the {COLUMNSTORE}. To modify data in the
+   {COLUMNSTORE}, use standard SQL.
+
+1. **View the policies that you set or the policies that already exist**
+
+   ``` sql
+   SELECT * FROM timescaledb_information.jobs
+   WHERE proc_name='policy_compression';
+   ```
+   See [timescaledb_information.jobs][informational-views].
+
+You can also [convert_to_columnstore][convert_to_columnstore] and [convert_to_rowstore][convert_to_rowstore] manually
+for more fine-grained control over your data.
+
+## Limitations
+
+{CHUNK}s in the {COLUMNSTORE} have the following limitations:
+
+*   `ROW LEVEL SECURITY` is not supported on {CHUNK}s in the {COLUMNSTORE}.
+
+## Available functions
+
+### Policies
+
+-  [`add_columnstore_policy()`][add_columnstore_policy]: set a policy to automatically move {CHUNK}s in a {HYPERTABLE}
+  to the {COLUMNSTORE} when they reach a given age
+- [`remove_columnstore_policy()`][remove_columnstore_policy]: remove a {COLUMNSTORE} policy from a {HYPERTABLE}
+
+### Configuration
+
+- [`ALTER TABLE (hypercore)`][alter_table_hypercore]: enable the {COLUMNSTORE} for a {HYPERTABLE}
+
+### Manual conversion
+
+- [`convert_to_columnstore()`][convert_to_columnstore]: manually add a {CHUNK} to the {COLUMNSTORE}
+- [`convert_to_rowstore()`][convert_to_rowstore]: move a {CHUNK} from the {COLUMNSTORE} to the {ROWSTORE}
+
+### Statistics and information
+
+- [`chunk_columnstore_stats()`][chunk_columnstore_stats]: get statistics about {CHUNK}s in the {COLUMNSTORE}
+-  [`hypertable_columnstore_stats()`][hypertable_columnstore_stats]: get {COLUMNSTORE} statistics related to the
+  {COLUMNSTORE}
+-  [`timescaledb_information.chunk_columnstore_settings`][chunk_columnstore_settings]: get information about settings on
+  each {CHUNK} in the {COLUMNSTORE}
+-  [`timescaledb_information.hypertable_columnstore_settings`][hypertable_columnstore_settings]: get information about
+  {COLUMNSTORE} settings for all {HYPERTABLE}s
```

Comment:
the links to available functions on this page don't work - I assume that's because they have currentversion in them